runtime/shared_common/j9shr.tdf

```diff
@@ -2945,3 +2945,17 @@ TraceEntry=Trc_SHR_OSC_getCacheVersionAndGen_Entry_V1 NoEnv Overhead=1 Level=1 T
 TraceEntry=Trc_SHR_CC_initializeWithCommonInfo_Entry2 NoEnv Overhead=1 Level=3 Template="CC initializeWithCommonInfo: memForConstructor=%p, ctrlDirName=%s, cacheName=%s, newPersistentCacheReqd=%d, layer=%d"
 TraceEntry=Trc_SHR_CC_initialize_Entry2 NoEnv Overhead=1 Level=1 Template="CC initialize: memForConstructor=%p, sharedClassConfig=%p, cacheName=%s, newPersistentCacheReqd=%d, layer=%d, unitTest=%zd"
 TraceEntry=Trc_SHR_OSC_newInstance_Entry_V1 NoEnv Overhead=1 Level=1 Template="SH_OSCache::newInstance: Entered, memForConstructor=%p, cacheName=%s, newPersistentCacheReqd=%d layer=%d"
+
+TraceException=Trc_SHR_CM_getPrereqCache_InvalidType Overhead=1 Level=1 Template="CM::getPrereqCache(): Invalid data type found in the shared cache at %p. The shared cache is corruptted"
+TraceEvent=Trc_SHR_CM_getPrereqCache_Found Overhead=1 Level=6 Template="CM::getPrereqCache(): Found prerequisite cache with unique ID %.*s"
+TraceEvent=Trc_SHR_CM_getPrereqCache_NotFound Overhead=1 Level=1 Template="CM::getPrereqCache(): This shared cache does not have a prerequisite cache"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Start_Manager Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to startup the corresponding manager"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Add_ID Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to add unique ID of prerequisite cache"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Get_Manager Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to get the corresponding manger"
+TraceException=Trc_SHR_CM_getPrereqCache_Failed_To_Store_Prereq_UniqueID Overhead=1 Level=1 Template="CM::getPrereqCache(): Failed to store the prerequisite cache unique ID %.*s"
+TraceException=Trc_SHR_CC_verifyCacheUniqueID_Failed Overhead=1 Level=1 Template="CC verify Cache Unique ID Failed. The Expected ID is %s, the current ID is %s"
+TraceExit=Trc_SHR_CM_startup_Exit8 Overhead=1 Level=1 Template="CM startup: Cache Unique ID Verification Failed"
```

Comment:
Seems all these Exit tracepoints should be TraceExit-Exception

Reply:
Fixed